errbot/backends/irc.py

```diff
@@ -481,8 +539,61 @@ def on_join(self, connection, event):
         # We need to wait to endofnames message.
         room_name = event.target
         with self._recently_joined_lock:
+            if room_name not in self.bot._rooms:
+                self.bot._rooms[room_name] = IRCRoom(room_name, self.bot)
             self._recently_joined_to.add(room_name)
 
+    def on_currenttopic(self, connection, event):
+        """
+            When you Join a room with a topic set this event fires up to
+            with the topic information.
+            If the room that you join don't have a topic set, nothing happens.
+            Here is NOT the place to fire the :meth:callback_room_topic event, for
+            that case exist on_topic.
+
+            :param:
+                connection: Is an 'irc.client.ServerConnection' object
```

Comment:
This param syntax is wrong, these should go on the same line, as in `:param connection:` otherwise Sphinx will fail to parse it.